events/templates/create_event.html

```diff
@@ -2,9 +2,9 @@
 {% load crispy_forms_tags %}
 {% block content %}
 <h2>{{note}}</h2>
-<form action="{%url 'create event'%}" method="post">
+<form action="" method="POST">
   {% csrf_token %}
   {{ form|crispy }}
-  <input type="submit" name="" value="Create">
+  <input type="submit" name="" value="Save">
```

Comment:
This form has no action? Is it because it's a model form?

Reply:
Needed so edit could inherit from this from and doesn't create a whole new event. Still functions for creating new events